extensions/scheduler/deployment/src/main/java/io/quarkus/scheduler/deployment/devui/SchedulerDevUIProcessor.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.scheduler.deployment.devui;
```

Comment:
Probably a stupid question but we don't have to remove anything from the old UI?

Or you plan to clean that up later?

Reply:
I have no idea what the plan is but it looks like we keep the old UI for now. CC @phillip-kruger

Reply:
OK I thought we would remove the old UI as we add to the new as, otherwise, it will be extra work in dev mode.

But that can be done in the end if we think it's better.

Reply:
Yes we will remove the old Dev UI once the new one is usable